lib/tools/adb-commands.js

```diff
@@ -674,4 +674,31 @@ methods.getDeviceProductLocale = async function () {
   return await this.getDeviceProperty("ro.product.locale");
 };
 
+methods.getModel = async function () {
+  let stdout = await this.shell('getprop');
```

Comment:
Why not be more specific, so the regexp is not necessary? `adb shell getprop ro.product.model` will just return the model name, no?